src/views/PopupView.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from "react";
+import * as cameraControlsImg from "../../assets/images/camera-controls.png";
+import * as selectionControlsImg from "../../assets/images/selection-controls.png";
+
+interface Props {
+    error: Error|null,
+    menu: boolean,
+    sdpInfo: boolean,
+
+    controlsHelpImgNum: number,
```

Comment:
doesn't appear to be used outside this component, so it seems like it would be better to move it inside

Reply:
Ah, good idea. Fixed.